src/features/island/hud/components/deliveries/TravelModal.tsx

```diff
@@ -32,44 +48,74 @@ export const TravelModal: React.FC<Props> = ({ isOpen, onClose }) => {
         onShow={() => gameService.send("SAVE")}
         dialogClassName="md:max-w-3xl"
       >
-        <CloseButtonPanel
-          onClose={onClose}
-          tabs={[
-            { icon: world, name: "Travel" },
-            { icon: SUNNYSIDE.icons.heart, name: "Deliveries" },
-            { icon: SUNNYSIDE.icons.expression_chat, name: "Chores" },
-          ]}
-          currentTab={tab}
-          setCurrentTab={setTab}
-        >
-          <div
-            style={{ maxHeight: CONTENT_HEIGHT }}
-            className="w-full pr-1 pt-2.5 overflow-y-auto scrollable"
+        {showIntro ? (
+          <Panel bumpkinParts={NPC_WEARABLES["reelin roy"]}>
+            <SpeakingText
+              message={[
+                {
+                  text: "Howdy Traveller, ready to set sail?",
```

Comment:
It felt like it was missing a bit of context before jumping into the Pumpkin Plaza.

We will add some level gating which means they can only initially visit the Plaza (and won't have other choices). The labels felt very clear to me, so I don't think we need to describe what is at the plaza or beach.

I would suggest the following:

`Howdy Traveller, ready to set sail?`

`Sunflower Land is filled with exciting islands where you can complete deliveries, craft rare NFTs and even dig for treasure!`

`Different locations bring different opportunities to spend your hard earned resources. At any time click the travel button to return home.`